src/almanac/software-forest/software-forest.md

```diff
@@ -0,0 +1,30 @@
+# Software Forest
+
+> "Called Three-Dimensional Forestry or Forest Farming, … the three 'dimensions' of \[Toyohiko Kagawa's\] 3-D system were the trees as conservers of the soil and suppliers of food and the livestock which benefited from them." {cite:p}`hart_forest_1996`, {cite:p}`kagawa_fujisaki_3d_farming`
+
+Forest gardening offers innovations from an ecological cultivation perspective by caring for trees, understory vegetation, soil, and other  environmental factors together.
+The software forest is a multi-layered forest garden enriched through connections and minimal, iterable change.
+Software forests are created gradually, "… starting with the existing, living, breathing system, and working outward, a step at a time, in such a way as to not undermine the system’s viability." {cite:p}`foote1997big`
+Nourishing the system in this way helps us practice _primum non nocere_ ("first, do no harm") to the garden and components within, conserving energy for necessary adaptations.
+
+Dense networks of relationships surround software forest gardens in the form of software environments.
+Some of these environments are portable, similar to [Wardian cases](https://en.wikipedia.org/wiki/Wardian_case) that provide loosely coupled habitats for software organisms (for example, through environment managers).
+Others are exhibited through abstract relationships as [_nurse logs_](https://en.wikipedia.org/wiki/Nurse_log), [_Hügelkultur_](https://en.wikipedia.org/wiki/H%C3%BCgelkultur), or [mycorrhizal](https://en.wikipedia.org/wiki/Mycorrhiza)-like associations which redistribute energy among the entirety of a software region.
+
+As software gardeners we must consider how energy is transferred to various portions of the forest.
+Adventitious code, which "volunteers" by accident of growth, may create scenarios of energy waste (also ) in the form of cognitive or performance misalignment.
```

Comment:
Not sure what is going on with the `(also )`?

Reply:
Thanks, I'll remove.